web/src/modules/layout/components/Header.js

```diff
@@ -92,7 +92,6 @@ const NavigationBar = styled.div`
   flex: 1 1 0%; /* The % unit is necessary for IE11 */
   align-items: stretch;
   justify-content: center;
-  overflow-y: hidden;
```

Comment:
AFAICS (according to git blame) this line was added to hide the scrollbar. Do you think there is another solution to this problem?

Reply:
I found this snipped to hide scrollbars but keep functionality.

maybe that works better for some weird css reason:

```
 /* Hide scrollbar for Chrome, Safari and Opera */
.example::-webkit-scrollbar {
  display: none;
}

/* Hide scrollbar for IE, Edge and Firefox */
.example {
  -ms-overflow-style: none;  /* IE and Edge */
  scrollbar-width: none;  /* Firefox */
} 
```

Reply:
I added the code Max suggested, but i can't see any difference. Tested on Safari on iOS, local Chrome-Browser and in the Android Emulator.
Which solution do you prefer / should we merge or rather do you see any difference?

Reply:
If I remember correctly, some scroll bar was shown on android... Not sure though

Reply:
Thanks for the hint with firefox. Tested this on ios and android emulator, but there is also not scrollbar.

![ios firefox](https://user-images.githubusercontent.com/18463686/104577010-42ef7700-5659-11eb-9b41-ab9e86a2d2aa.jpg)
![android firefox](https://user-images.githubusercontent.com/18463686/104577033-48e55800-5659-11eb-8bee-b9a7cd737474.png)

Reply:
Could also not reproduce any scrollbar. @klinzo how old was the change with the scrollbar? Maybe we fixed it sometime in between.

Reply:
Okay, nice :+1: I just used firefox and set dimension to an iPhone :) But I guess this is fine then.